custom_completions.go

```diff
@@ -132,6 +132,12 @@ func (c *Command) initCompleteCmd(args []string) {
 					comp = strings.Split(comp, "\t")[0]
 				}
 
+				// Make sure we only write the first line to the output.
+				// This is needed if a description contains a linebreak.
+				// Otherwise the shell scripts will interpret the other lines as new flags
+				// and could therefor provide a wrong suggestion.
```

Comment:
Typo: "therefore"

Reply:
Could you change "suggestion" to "completion"?  Cobra has a concept of "suggestions" when someone mistypes a command; it is different from "completions".

Reply:
OK, thanks.